lib/datasource/sbt-package/index.spec.ts

```diff
@@ -30,20 +30,74 @@ describe('datasource/sbt', () => {
       nock.disableNetConnect();
       nock('https://failed_repo').get('/maven/org/scalatest/').reply(404, null);
       nock('https://repo.maven.apache.org')
+        .get('/maven2/com/example/')
+        .reply(200, '<a href="empty/">empty_2.12/</a>\n');
+      nock('https://repo.maven.apache.org')
+        .get('/maven2/com/example/empty/')
+        .reply(200, '');
+      nock('https://repo.maven.apache.org')
+        .persist()
```

Comment:
```suggestion
```

Reply:
This one is necessary because we have multiple `expect`s that make requests to the same group ID in [a single `it` clause](https://github.com/renovatebot/renovate/pull/6756/files#diff-bac525c841a27b039da8315a428e0146R204) but `nock` allows only 1 request without `persist()` or `times()`. 

We have the following options.

- (a) leave `persist()`
- (b) replace it with `times(3)` (placed after `get()`)
- (c) separate `it` clause according to the conditions of existence/absense of `homepage` or `sourceUrl`
- (d) separate group IDs

Which one do you think is preferable?

Reply:
Please use **(b)** so the tests are more predictable.

Reply:
Fixed in 70c06d2